Makefile

```diff
@@ -15,11 +15,14 @@ all: release
 count = 1
 # pkgs changes which packages the makefile calls operate on. run changes which
 # tests are run during testing.
-pkgs = ./ ./api ./database ./skyd
+pkgs = ./ ./api ./database ./skyd ./test ./test/workers
```

Comment:
Not sure if I should have put the missing packages here  or in `integration-pkgs`.

Reply:
I would add them to `integration-pkgs` and then include `integration-pkgs` here. This is seemilar to how we handle the `lockcheck` packages in `skyd`.

fine as a follow up to clean up this file.

Reply:
https://github.com/SkynetLabs/pinner/issues/31